shared/checks/oval/bootloader_password.xml

```diff
@@ -11,7 +11,10 @@
     <criteria operator="OR">
       <criterion comment="Pass if /boot/grub2/grub.cfg does not exist" test_ref="test_bootloader_grub_cfg" />
       <criteria operator="AND">
-        <criterion comment="make sure a password is defined in /boot/grub2/user.cfg" test_ref="test_bootloader_password" />
+        <criteria comment="check both files to account for procedure change in documenation" operator="OR">
```

Comment:
The password should be configured in both places, right?
So operator should be `AND`.

Reply:
I was confused, never mind.